consensus/api/proto/blockchain.proto

```diff
@@ -38,6 +38,7 @@ message Block {
     uint32 version = 2;
     bytes parent_id = 3;
     uint64 index = 4;
+    uint64 cumulative_txo_count = 7;
```

Comment:
It's confusing to me that the id numbers are not ordered in an incremental order. I'm not sure if it's better this way, or if this field should be put after `contents_hash`

Reply:
And if we're doing the breaking change mentioned above, we'll be able to just change the tag numbers.